README.md

```diff
@@ -39,18 +39,30 @@ built from source using the latest version of [Go](https://golang.org), and the
 available instructions in our
 [Wiki](https://github.com/git-lfs/git-lfs/wiki/Installation#source).
 
-### Usage
+#### Installing from a [binary package](https://github.com/git-lfs/git-lfs/releases)
 
-Git LFS requires a global installation once per-machine. This can be done by
+The included script will install Git LFS binaries. It also runs `git lfs install` to
+perform required global configuration changes.
+
+```bash
+$ ./install.sh
+```
+
+#### Installing manually
+
+- Place the `git-lfs` binary on your system’s executable `PATH` or equivalent.
```

Comment:
Let's refer to `$PATH` as such (with the proceeding `
$`) to indicate that it is an environment variable.